### PR TITLE
Added support for markdown in description plugin and live metrics

### PR DESCRIPTION
**Pull request by andriichumak**
Based on request from enablement team.

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: dashboard_description_plugin/webpack.config.js [hunk 1]
```diff
@@ -165,7 +165,7 @@ module.exports = (_env, argv) => {
             plugins: [
                 ...commonConfig.plugins,
                 new ProvidePlugin({
-                    process: "process/browser",
+                    process: "process/browser.js",
```
Comment (no23reason) [inline dashboard_description_plugin/webpack.config.js:168]: Why is this change needed? Should we update the template for other plugins as well?

Comment (andriichumak) [reply dashboard_description_plugin/webpack.config.js:168]: This was needed due to the way`react-markdown` is packaged, it would not compile otherwise.
```
ERROR in ./node_modules/uvu/node_modules/kleur/index.mjs 4:11-18
  Module not found: Error: Can't resolve 'process/browser' in '.../gooddata-plugin-examples/dashboard_description_plugin/node_modules/uvu/node_modules/kleur'
  Did you mean 'browser.js'?
  BREAKING CHANGE: The request 'process/browser' failed to resolve only because it was resolved as fully specified
  (probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
  The extension in the request is mandatory for it to be fully specified.
  Add the extension to the request.
```

Comment (no23reason) [reply dashboard_description_plugin/webpack.config.js:168]: Ok, thanks for the explanation :)

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- gooddata-plugin-examples-fossa-scanning-zuul-el7 : SKIPPED (non-voting)
- gooddata-plugin-examples-sonarqube-scan-zuul-gate : SKIPPED
- https://checklist.intgdc.com/job/compliance/job/gooddata-plugin-examples-merge-compliance-check/20/ : FAILURE in 21s

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-plugin-examples-gate-PR21

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.
